newrelic/newrelic.go

```diff
@@ -84,6 +80,12 @@ func Middleware(next http.Handler) http.Handler {
 	})
 }
 
+func Shutdown(duration time.Duration) {
+	if app != nil {
+		app.Shutdown(duration)
+	}
+}
+
```

Comment:
This isn't used anywhere? Is it for handling the errors in the service?

Reply:
@jeremy-homes  We use newrelic, but if a service is stopped or panic, we don't give newrelic time to send last messages. It's like `flush`. 

I want to get rid of all panics and make sure that we do a proper server shutdown and finish all tasks (close connections, send message to bugnag or newrelic)